python/interpret/ext/glassbox/test/test_example_glassbox_explainer_ext.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2019 Microsoft Corporation
+# Distributed under the MIT software license
+
+from ....glassbox import LinearRegression
+from ....api.base import ExplainerMixin
+from .. import _is_valid_glassbox_explainer
+
+
+def test_invalid_glassbox_explainer():
```

Comment:
I get that this is a bit more difficult, but can we at least test that one of the current glassbox explainers in the interpret package will pass our checks?

Reply:
sure, I can make a separate PR with that test, would like to get this in first